azafea/event_processors/endless/metrics/v2/model/_machine.py

```diff
@@ -17,7 +17,7 @@
 
 from azafea.model import Base, DbSession
 
-from ..image import parse_endless_os_image
+from ....image import parse_endless_os_image
```

Comment:
Wow, I had no idea you could just keep adding `.` to traverse further up the hierarchy.

Reply:
Original idea from the Python devs. That’s both ugly and useful in my opinion 😄.